src/borg/helpers.py

```diff
@@ -645,6 +650,16 @@ def replace_placeholders(text):
     return format_line(text, data)
 
 
+HUMAN_SORT_KEYS = ['timestamp'] + list(ArchiveInfo._fields)
+HUMAN_SORT_KEYS.remove('ts')
+
+def sort_by_spec(text):
+    for token in text.split(','):
+        if token not in HUMAN_SORT_KEYS:
+            raise ValueError('Invalid sort key: %s' % token)
+    return text.replace('timestamp', 'ts')
+
+
 def safe_timestamp(item_timestamp_ns):
     try:
         return datetime.fromtimestamp(bigint_to_int(item_timestamp_ns) / 1e9)
```

Comment:
maybe do all the fixups into 1 fixup commit for this commit, so you can git rebase -i later and squash the fixup commit into the base commit.